### ISSUE TITLE
Add replication test endpoint

### ISSUE THREAD

**Issue by mlissner** [freelawproject/courtlistener#1324]
This is actually pretty easy. Once/minute, write to a field using the default DB, then read the field's value on the replicated DB. If the difference grows too large, we're in trouble. Track the rolling average if we want to be fancy. Maybe just in a log, maybe by logging errors to Sentry.

**Comment by mlissner** [comment]
Oh, and if we want to get really fancy, we could add an endpoint for this where customers and others can check the status.

**Comment by mlissner** [comment]
A better way to do this might be to run a query like this on our publishers:

```
SELECT 
  slot_name,
  confirmed_flush_lsn, 
  pg_current_wal_lsn(), 
  (pg_current_wal_lsn() - confirmed_flush_lsn) AS lsn_distance
FROM pg_replication_slots;
```

I think that'll give us the distance that it's lagging. If it gets above a certain threshold, we can start sending alarms, just like AWS does.

**Comment by mlissner** [comment]
Fixed in 0188cba362639918e4cbdde0e61c6d79af00b13e
